### PR TITLE
switching to stripes customer portal

### PR DESCRIPTION
**Pull request by PeerRich**
current customer portal seems to be broken for some. we're switching to stripes native customer portal. wip, don't merge, yet.

### CODE REVIEW

**Comment by zomars** [comment]
Implemented, but there some quirks from Stripe side. Opened a support ticket and awaiting response.

image

**Comment by PeerRich** [comment]
> Implemented, but there some quirks from Stripe side. Opened a support ticket and awaiting response.
> 
> image

any responses?

**Comment by zomars** [comment]
I’m afraid not yet

El 26 sep 2021, a la(s) 16:06, Peer Richelsen ***@***.***> escribió:

﻿
Implemented, but there some quirks from Stripe side. Opened a support ticket and awaiting response.


any responses?

**Comment by zomars** [comment]
I mean, the current billing portal doesn't work either. So let's test on staging and wait for an official Stripe response.

File: ee/pages/api/integrations/customer-portal.ts
Comment (emrysal) [inline ee/pages/api/integrations/customer-portal.ts:29]: Simple fix, I suppose.

**Comment by PeerRich** [comment]
> I mean, the current billing portal doesn't work either. So let's test on staging and wait for an official Stripe response.

same issue 
image

**Comment by PeerRich** [comment]
regardless of that issue, I would change this page:

image


to open the billing portal of stripe when clicking the "Billing" tab vs showing the button

**Review by emrysal — DISMISSED** [review]
Looks good to me, but it also looks like a force push because lint seems all over the place. In any case you probably intended to use that stripeCredentials variable so that is my only change request.

**Comment by zomars** [comment]
> regardless of that issue, I would change this page:
> 
> image
> 
> to open the billing portal of stripe when clicking the "Billing" tab vs showing the button
> 
> essentially opening the stripe customer portal by clicking the tab

Since this is an external link, I think it would be confusing if a tab takes me to an external page. I would be even more explicit with the button text and put something like "Manage billing in Stripe's website" or something like that.

**Comment by baileypumfleet** [comment]
> Since this is an external link, I think it would be confusing if a tab takes me to an external page. I would be even more explicit with the button text and put something like "Manage billing in Stripe's website" or something like that.

I agree with this. Once it's merged I can make the UI a bit better but I think it makes sense to not make the tab a button.

Once @emrysal's change is fixed I'm happy to approve & merge

**Comment by zomars** [comment]
All ready @baileypumfleet

**Pull request merged**
